package.json

```diff
@@ -10,7 +10,8 @@
     "lint": "node_modules/.bin/eslint ./ || true",
     "test": "nyc --reporter=html --reporter=text mocha --require babel-core/register ./src/tests/ || true",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "report-coverage": "nyc --reporter=lcov --reporter=text-lcov npm test"
+    "report-coverage": "nyc --reporter=lcov --reporter=text-lcov npm test",
+    "start": "node --require babel-core/register server/server"
```

Comment:
you would notice here that i didn't use node_modules. I didn't because i installed express using npm i express -S (admin permission reasons) and it didnt show in the node_modules so it wont work that way. However it works this way though.

Reply:
Try restructuring your start script to read:

`"start": "babel-node server/server"`

Does it work?

Reply:
No it doesn't.  Error: Unhandled 'error' event

Reply:
I just did this, and it worked.